languages/julia/generic/Parse_julia_tree_sitter.ml

```diff
@@ -1853,7 +1853,14 @@ and map_quotable (env : env) (x : CST.quotable) : expr =
             v2)
           v3
       in
-      let base = Seq (v2 :: v3) |> G.e in
+      let base =
+        match v3 with
+        (* This means we would produce a singleton Seq. Let's not do that, and just
+           take the expression itself.
+        *)
+        | [] -> v2
```

Comment:
you could return also a ParenExpr. You could also just return v2 but set its range
to also cover the opening and closing parenthesis for autofixing to work better on such code.